### DISCUSSION TITLE
Hide members when no `members` option

### DISCUSSION THREAD

**Discussion by femtomc** [mkdocstrings/python#101] [Q&A] [answered]
From the recent update, the following:

```
::: some_mod.Class
```

now renders all members (previously, I think it would just print the "root level" doc annotation. Is it possible to turn this behavior off?

**Comment by pawamoy** [comment, marked as answer]
If it didn't previously show members, it was a bug. To turn off rendering members, you can use this:

```
::: some_mod.Class
    options:
      members: false
```
